lib/metadata_json_lint.rb

```diff
@@ -135,6 +135,11 @@ def invalid_requirements?(requirements)
   module_function :invalid_requirements?
 
   def invalid_dependencies?(deps)
+    unless deps.is_a?(Array)
+      puts "Warning: Dependencies must be in an array. Currently it's a #{deps.class}."
+      error_state = true
```

Comment:
Or, conversely, if you raise it from Warning to Error in the description on L139.